src/microbe_stage/Microbe.cs

```diff
@@ -531,6 +533,13 @@ public override void _IntegrateForces(PhysicsDirectBodyState physicsState)
         // TODO: should movement also be applied here?
 
         physicsState.Transform = GetNewPhysicsRotation(physicsState.Transform);
+
+        collisionForce = Vector3.Zero;
+
+        for (var i = 0; i < physicsState.GetContactCount(); ++i)
+        {
+            collisionForce += physicsState.GetContactImpulse(i) * physicsState.GetContactLocalNormal(i);
```

Comment:
Does this mean that, if a microbe impacts two objects on opposite sides at the same time (like squeezing through a gap), that it will have near zero collision force?

Reply:
Hmm, I actually haven't thought about this. I will look into this later when I have more time.

Reply:
I would imagine that results in two different contacts as they are different objects that hit the one in the center.

Reply:
And also at least in bullet two objects colliding can result in multiple contact points depending on their shapes.

Reply:
Ah, I see these are summed here. Maybe the squared lengths should already be retrieved here so that this problem won't be able to happen.